Mixpanel/MPNotificationViewController.m

```diff
@@ -264,13 +266,6 @@ - (void)viewDidLoad {
         } else {
             self.imageView.hidden = YES;
         }
-
-        if (notification.body != nil) {
```

Comment:
because notification.body shouldn't ever be nil in minis. and if it is, then we dont even present the notification anymore